docs/download-apis.js

```diff
@@ -3,41 +3,54 @@
  * Licensed under the MIT License.
  */
 
-/**
+/*
```

Comment:
As a jsdoc comment, VSCode wants to apply the docs to the `chalk` import, rather than just treating it as a module-level comment.